.github/workflows/npm.yml

```diff
@@ -0,0 +1,53 @@
+name: NPM
```

Comment:
We need to update this workflow to something similar as in https://github.com/keep-network/coverage-pools/pull/186, especially add `Deploy contracts` step.